### ISSUE TITLE
Check if pings can be spammed

### ISSUE THREAD

**Issue by Aivean** [spaceshelter/orbitar#115]
`@` mentions in posts and comments create a record in DB. Verify that this cannot be abused.

**Comment by 4vanger** [comment]
Still actual after you deduped the notifications?

**Comment by Aivean** [comment]
> Still actual after you deduped the notifications?

Yep, possible to mention lots of different people, and by spamming comments.
